doc.go

```diff
@@ -147,13 +147,6 @@
 // the handshake if the Origin request header is present and not equal to the
 // Host request header.
 //
-// An application can allow connections from any origin by specifying a
-// function that always returns true:
-//
-//  var upgrader = websocket.Upgrader{
-//      CheckOrigin: func(r *http.Request) bool { return true },
```

Comment:
Add additional context to the `CheckOrigin` field in the `Upgrader` struct? https://godoc.org/github.com/gorilla/websocket#Upgrader

from:
```
    // CheckOrigin returns true if the request Origin header is acceptable. If
    // CheckOrigin is nil, the host in the Origin header must not be set or
    // must match the host of the request.
```

to:
```
    // CheckOrigin returns true if the request Origin header is acceptable. If
    // CheckOrigin is nil, the host in the Origin header must not be set or
    // must match the host of the request. You should carefully valid
    // Origin headers, and not blindly accept requests from any origin.
```

Reply:
@elithrar Please comment on my most recent change.